### PR TITLE
feat: install scripts without a database 

### LINKED ISSUE #141: Install scripts without a database
## Is your feature request related to a problem? Please describe.

Compared to plugins, scripts are more numerous. Therefore, it will take a lot of time for all the data to be entered into the database.

## Describe the solution you'd like

Add a way to install the script without a database. Fortunately, the way to install scripts is simpler than plugins. I think you can do the installation programmatically.

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Supports the installation of scripts that are not included in the database.

List of things to do before merging
- Create and publish an HTML file containing a link to the script
- Consider a better name for the button.

## Related Issue

closes #141

## How Has This Been Tested?

- You can install scripts that are not listed.
- You can uninstall installed scripts
- You can update from an existing version of APM without problems

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by mitosagi** [comment]
More buttons will be added to the UI. Any ideas on how to make them more compact?

This feature requires [a set of links to the script](https://mitosagi.github.io/apm-test-ghpage/apm-simple-install.html).
I would like to create an html folder in apm-data and publish it on github pages. Is this possible?

**Comment by hal-shu-sato** [comment]
> More buttons will be added to the UI. Any ideas on how to make them more compact?

I think so too.
I have an idea using the dropdown but it may make the buttons difficult to use.

> This feature requires [a set of links to the script](https://mitosagi.github.io/apm-test-ghpage/apm-simple-install.html).
> I would like to create an html folder in apm-data and publish it on github pages. Is this possible?

Nice page.
Yes, you can.
If you want to use GitHub Pages, the directory name should be `docs`.

File: src/package/package.js [hunk 1]
```diff
@@ -707,8 +719,17 @@ module.exports = {
     }
 
     if (filesCount === existCount) {
-      apmJson.removePackage(instPath, uninstalledPackage);
-      await this.setPackagesList(instPath, true);
+      if (!uninstalledPackage.id.startsWith('script_')) {
+        apmJson.removePackage(instPath, uninstalledPackage);
+        await this.setPackagesList(instPath, true);
+      } else {
+        apmJson.removePackage(instPath, uninstalledPackage);
```
Comment (hal-shu-sato) [inline src/package/package.js:726]: There are the same processes.

```suggestion
      apmJson.removePackage(instPath, uninstalledPackage);
      if (!uninstalledPackage.id.startsWith('script_')) {
        await this.setPackagesList(instPath, true);
      } else {
```

File: src/preload.js [hunk 2]
```diff
@@ -97,6 +97,15 @@ window.addEventListener('load', () => {
     await package.installPackage(installPackageBtn, installationPath.value);
   });
 
+  const installScriptBtn = document.getElementById('install-script');
+  installScriptBtn.addEventListener('click', async (event) => {
+    await package.installScript(
+      installScriptBtn,
+      installationPath.value,
+      'https://mitosagi.github.io/apm-test-ghpage/apm-simple-install.html'
```
Comment (hal-shu-sato) [inline src/preload.js:105]: Perhaps is this a test page?

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
There are some points to check.

File: src/preload.js (hunk 2 above)
Comment (mitosagi) [reply src/preload.js:105]: I'll replace it later.

**Comment by mitosagi** [comment]
I fixed it.

**Review by hal-shu-sato — APPROVED** [review]
Good feature!
As you said, let's think about the design.

**Pull request merged**